uint8 id
---
int8 temperature
